packages/components/src/Navbar/Component.purs

```diff
@@ -179,7 +180,7 @@ customerService isPersonating activeUser = do
       userLink user =
         Router.link
           { to: { pathname: "/", state: {} }
-          , children: [ DOM.text $ user.cusno <> " - " <>
+          , children: [ DOM.text $ (show $ (\(Cusno c) -> c) $ user.cusno) <> " - " <>
```

Comment:
I think `show $ (\(Cusno c) -> c` could become a `toString` or `fromCusno` function to put in the `KSF.User.Cusno` module.

Reply:
I added `toString`. Using plain `show` in code is always suspicious anyway.